### ISSUE TITLE
The faces of Asian people in the processed photos have Western facial features after high-definition

### ISSUE THREAD

**Issue by BluesWhale** [sczhou/CodeFormer#117]
尝试高清处理老照片后，照片中的亚洲人物面相变得有欧美人面相特征 

The faces of Asian people in the processed photos have Western facial features after high-definition

**Comment by sczhou** [comment]
Could you post the test image here? If the image is very blurry, it may result in the loss of personal features and make it difficult for face restoration models, including CodeFormer, to accurately restore personal characteristics. This is due to the data bias in the FFHQ training dataset (commonly used in this field), which primarily includes Caucasian faces.

**Comment by DanvilleC** [comment]
here is an example, asian baby looks like a barby doll after resoration:
![image](https://user-images.githubusercontent.com/125329172/219303013-1ccb974b-ac18-4e83-94e2-28937e484221.png)
![image](https://user-images.githubusercontent.com/125329172/219302865-36c9e897-c126-4c02-8e2a-bd0e25029fd9.png)
